src/frontend/utils/amplitude.ts

```diff
@@ -22,76 +22,49 @@ export function logEvent(eventName: string, eventProperties: any) {
     amplitudeInstance.logEvent(eventName, eventProperties);
 }
 
-export const logSidevisningBarnetrygd = (side: string) => {
+export const logSidevisningBarnetrygd = (side: string, søknadstype: ESøknadstype) => {
     logEvent('sidevisning', {
         side,
         team_id: 'familie',
-        skjemanavn: søknadstyper[hentSøknadstype()].navn,
-        skjemaId: søknadstyper[hentSøknadstype()].id,
+        skjemanavn: søknadstyper[søknadstype].navn,
+        skjemaId: søknadstyper[søknadstype].id,
     });
 };
 
-export const logSkjemaStartet = () => {
+export const logSkjemaStartet = (søknadstype: ESøknadstype) => {
     logEvent('skjema startet', {
-        skjemanavn: søknadstyper[hentSøknadstype()].navn,
-        skjemaId: søknadstyper[hentSøknadstype()].id,
+        skjemanavn: søknadstyper[søknadstype].navn,
+        skjemaId: søknadstyper[søknadstype].id,
         team_id: 'familie',
     });
 };
 
-export const logForsettPåSøknad = () => {
+export const logFortsettPåSøknad = (søknadstype: ESøknadstype) => {
     logEvent('fortsett på søknad', {
-        skjemanavn: søknadstyper[hentSøknadstype()].navn,
-        skjemaId: søknadstyper[hentSøknadstype()].id,
+        skjemanavn: søknadstyper[søknadstype].navn,
+        skjemaId: søknadstyper[søknadstype].id,
         team_id: 'familie',
     });
 };
 
-export const logSkjemaStegFullført = (steg: number) => {
+export const logSkjemaStegFullført = (steg: number, søknadstype: ESøknadstype) => {
     logEvent('skjemasteg fullført', {
-        skjemanavn: søknadstyper[hentSøknadstype()].navn,
-        skjemaId: søknadstyper[hentSøknadstype()].id,
+        skjemanavn: søknadstyper[søknadstype].navn,
+        skjemaId: søknadstyper[søknadstype].id,
         team_id: 'familie',
         steg,
     });
 };
 
-export const logKlikkGåVidere = (steg: number) => {
+export const logKlikkGåVidere = (steg: number, søknadstype: ESøknadstype) => {
     logEvent('klikk gå videre', {
-        skjemanavn: søknadstyper[hentSøknadstype()].navn,
-        skjemaId: søknadstyper[hentSøknadstype()].id,
+        skjemanavn: søknadstyper[søknadstype].navn,
+        skjemaId: søknadstyper[søknadstype].id,
         team_id: 'familie',
         steg,
     });
 };
 
-export const logSpørsmålBesvart = (spørsmålSpråktekstId: string, svar: string) => {
-    /**
-     * Vil ikke tulle med språkcontext, skal uansett ikke formatere spørsmålene (alle må være like uansett hva barnets navn er)
-     * derfor bruker vi språktekstene fa bokmålfila direkte uten intl.formatMessage
-     */
-    const spørsmål = bokmålTekster[spørsmålSpråktekstId] ?? false;
-
-    spørsmål &&
-        logEvent('skjemaspørsmål besvart', {
-            skjemanavn: søknadstyper[hentSøknadstype()].navn,
-            skjemaId: søknadstyper[hentSøknadstype()].id,
-            team_id: 'familie',
-            spørsmål,
-            svar,
-        });
-};
-
-export const logError = (error: Error) => {
```

Comment:
Kan ikke se at vi har noen visninger for denne i amplitude. Vi mener dessuten at det er unødvendig all den tid vi har feilovervåking i sentry